packages/serverpod/lib/src/server/endpoint_dispatch.dart

```diff
@@ -52,6 +52,41 @@ abstract class EndpointDispatch {
     return connector;
   }
 
+  Future<(MethodConnector, Map<String, dynamic>)> tryGetEndpointMethod({
```

Comment:
Suggestion: Can we find a different name for this method or move the auth logic? This method is not simply retrieving the endpoint as it is also executing the auth.

Reply:
I was thinking exactly the same but didn't end up at a future proof name. I was considering `tryAuthorizeAndGetEndpointMethod`, which would be more or less true for the current case. The potential problem I thought of was that since this will be the main code path for everything not related to the transport layer, other things might be added here in the future which doesn't fit the description. But maybe that's a future problem and it's very easy to rename so. WDYT?

Reply:
Renamed to `getAuthorizedEndpointMethodStreamConnector`